code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -76,6 +76,12 @@
 			else								//Everyone else fails, skip the emote attempt
 				return
 
+		if("tremble")
```

Comment:
This should absolutely not be a Diona only emote.

People can tremble, animals can tremble, it's hardly something unique to trees.

Reply:
Mmh it could just have a name change, something like creaks?

Reply:
Probably--something distinctively associated with trees/plants.